api/controllers/search.js

```diff
@@ -86,18 +86,22 @@ const searchCollection = async function (roles, keywords, schemaName, pageNum, p
     throw new Error('Search missing match aggregation');
   }
 
+  // keyword regex
+  let keywordRegexFilter = searchAggregator.createKeywordRegexAggr(decodedKeywords, schemaName);
+
   // Create the sorting and paging aggregations.
   const sortingPagingAggr = searchAggregator.createSortingPagingAggr(schemaName, sortingValue, sortField, sortDirection, pageNum, pageSize);
 
   // Combine all the aggregations.
   let aggregation;
   if (!schemaAggregation) {
-    aggregation = [...matchAggregation, ...sortingPagingAggr];
+    aggregation = [...matchAggregation, ...keywordRegexFilter, ...sortingPagingAggr];
   } else {
-    aggregation = [...matchAggregation, ...schemaAggregation, ...sortingPagingAggr];
-
+    aggregation = [...matchAggregation, ...schemaAggregation, ...keywordRegexFilter, ...sortingPagingAggr];
   }
 
+  console.log(JSON.stringify(aggregation));
```

Comment:
Can be removed.

Reply:
Whoops, checked in debug code. Will revert